README.md

```diff
@@ -2,15 +2,41 @@
 
 Nullstone Block standing up an AWS Fargate cluster with service discovery and execution role ready for launching container services.
 
+## Security & Compliance
+
+Security scanning is graciously provided by Bridgecrew. Bridgecrew is the leading fully hosted, cloud-native solution providing continuous Terraform security and compliance.
+
+[![Infrastructure Security](https://www.bridgecrew.cloud/badges/github/nullstone-modules/aws-fargate/general)](https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=nullstone-modules%2Faws-fargate&benchmark=INFRASTRUCTURE+SECURITY)
+[![CIS AWS V1.3](https://www.bridgecrew.cloud/badges/github/nullstone-modules/aws-fargate/cis_aws_13)](https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=nullstone-modules%2Faws-fargate&benchmark=CIS+AWS+V1.3)
+[![PCI-DSS V3.2](https://www.bridgecrew.cloud/badges/github/nullstone-modules/aws-fargate/pci)](https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=nullstone-modules%2Faws-fargate&benchmark=PCI-DSS+V3.2)
+[![NIST-800-53](https://www.bridgecrew.cloud/badges/github/nullstone-modules/aws-fargate/nist)](https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=nullstone-modules%2Faws-fargate&benchmark=NIST-800-53)
+[![ISO27001](https://www.bridgecrew.cloud/badges/github/nullstone-modules/aws-fargate/iso)](https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=nullstone-modules%2Faws-fargate&benchmark=ISO27001)
+[![SOC2](https://www.bridgecrew.cloud/badges/github/nullstone-modules/aws-fargate/soc2)](https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=nullstone-modules%2Faws-fargate&benchmark=SOC2)
+[![HIPAA](https://www.bridgecrew.cloud/badges/github/nullstone-modules/aws-fargate/hipaa)](https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=nullstone-modules%2Faws-fargate&benchmark=HIPAA)
+
 ## Inputs
 
+- `versioning: bool`
```

Comment:
These two inputs look like they are for S3. Did these accidentally get copied in?

Reply:
Good catch. Resolved.